r2i/onnxrt_openvino/parameters.cc

```diff
@@ -0,0 +1,216 @@
+/* Copyright (C) 2020 RidgeRun, LLC (http://www.ridgerun.com)
+ * All Rights Reserved.
+ *
+ * The contents of this software are proprietary and confidential to RidgeRun,
+ * LLC.  No part of this program may be photocopied, reproduced or translated
+ * into another programming language without prior written consent of
+ * RidgeRun, LLC.  The user is free to modify the source code after obtaining
+ * a software license from RidgeRun.  All source code changes must be provided
+ * back to RidgeRun without any encumbrance.
+ */
+
+#include "parameters.h"
+
+#include <string>
+
+namespace r2i {
+namespace onnxrt_openvino {
+
+Parameters::Parameters () {
+  ParamDesc logging_level_desc = {
+    {logging_level_meta},
+    std::make_shared<LoggingLevelAccessor>(this)
+  };
+  parameter_map.emplace(std::make_pair(logging_level_meta.name,
+                                       logging_level_desc));
+
+  ParamDesc log_id_desc = {
+    {log_id_meta},
+    std::make_shared<LogIdAccessor>(this)
+  };
+  parameter_map.emplace(std::make_pair(log_id_meta.name, log_id_desc));
+
+  ParamDesc intra_num_threads_desc = {
+    {intra_num_threads_meta},
+    std::make_shared<IntraNumThreadsAccessor>(this)
+  };
+  parameter_map.emplace(std::make_pair(intra_num_threads_meta.name,
+                                       intra_num_threads_desc));
+
+  ParamDesc graph_optimization_level_desc = {
+    {graph_optimization_level_meta},
+    std::make_shared<GraphOptLevelAccessor>(this)
+  };
+  parameter_map.emplace(std::make_pair(graph_optimization_level_meta.name,
+                                       graph_optimization_level_desc));
+
+  ParamDesc hardware_id_desc = {
+    {hardware_id_meta},
+    std::make_shared<HardwareIdAccessor>(this)
+  };
+  parameter_map.emplace(std::make_pair(hardware_id_meta.name, hardware_id_desc));
+}
+
+RuntimeError Parameters::Configure (std::shared_ptr<r2i::IEngine> in_engine,
+                                    std::shared_ptr<r2i::IModel> in_model) {
```

Comment:
here you have 2 different types of indentation aligning the second parameters line to the opening ( and 4 spaces indentation, I don't know what is the standard you are following but make it consistent